.env

```diff
@@ -82,6 +82,8 @@ TZ=UTC
 
 # -1 does not attempt to install a devtoolset version, any positive integer will install devtoolset-n
 DEVTOOLSET_VERSION=-1
+# set to true to enable ccache build configuration
+R_BUILD_CCACHE=false
```

Comment:
Remind me why we have to define this in .env and not just in the crossbow template?

Unrelated but the block above L75-81 has gotten confused. The lines that should follow the comment are L77, L78, L80; the others should be moved away from the comment. 

I also wonder if the env var should have a more clear name. Like, this is not just building with ccache, it's building it following the particular guidance of http://dirk.eddelbuettel.com/blog/2017/11/27/. As you point out in the other comment, we're still using ccache by default in the arrow build, regardless of the value here.

Reply:
_nods_ I'll work on a better name + see if we can do this in just crossbow

Reply:
One thing that will happen without it is for builds where it is not set, you get:

```
WARNING: The R_CUSTOM_CCACHE variable is not set. Defaulting to a blank string.
```

Which is fine, but is one consequence

Reply:
:/ so should we just add it to .env then? I feel like that warning will be confusing to someone running docker-compose locally, won't it?

Reply:
@jonkeane Please add `R_CUSTOM_CCACHE` to `.env` to suppress the warning. We should add the default value to `.env` when we add a new variable that is used in `docker-compose.yml`.